rkt/metadata_service.go

```diff
@@ -335,20 +340,25 @@ func handlePodAnnotation(w http.ResponseWriter, r *http.Request, pm *schema.PodM
 		return
 	}
 
-	w.Header().Add("Content-Type", "application/json")
+	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
```

Comment:
You're right - I'm just going by the validator program (and what the spec says). The spec is probably incorrect.